examples/sandbox/src/pages/ecommerce/Category.js

```diff
@@ -41,13 +41,14 @@ const categoryPageconfig = {
       }
     }
   }
-};
+});
 
-export default function CategoryPageTv() {
+export default function CategoryPage(props) {
+  const category = props?.match?.params.category;
   return (
     <>
       <Navigation />
-      <SearchProvider config={categoryPageconfig}>
+      <SearchProvider config={categoryPageconfig(category)}>
```

Comment:
This page has some TV-specific facets (below). In my PR, I also added some TV-specific fields in the config. If we go forward with a general category page, we should probably move those somewhere.